_maps/RandomRuins/SpaceRuins/deepstorage.dmm

```diff
@@ -0,0 +1,185 @@
+"aa" = (/turf/open/space,/area/space)
```

Comment:
use the template_noop instead of space. This means it wont load and change space to space.